source/speech/sayAll.py

```diff
@@ -104,8 +111,19 @@ def readText(
 		startPos: Optional[textInfos.TextInfo] = None,
 		nextLineFunc: Optional[Callable[[textInfos.TextInfo], textInfos.TextInfo]] = None,
 		shouldUpdateCaret: bool = True,
+		startedFromScript: bool | None = False,
 	) -> None:
+		"""Start or restarts the reader
```

Comment:
@seanbudd, @LeonarddeR, not sure that my Sphinx formatting of docstring is OK. As requested in the past, an example directly in NVDA's `codingStandards.md` file would be appreciated; the Sphinx tutorial that is linked instead does not match exactly our recommendations since the examples are with types.